README.md

```diff
@@ -73,6 +74,9 @@ public class SampleClient {
             try {
                 TransferStateApiResponse transferState = client.getTransferState(TransferStateApiRequest.newBuilder().setTransferId(transferApiResponse.getTransferId()).build());
                 System.out.println("Latest Transfer State " + transferState.getState());
+                if (transferState.getState().equals("COMPLETED")) {
```

Comment:
if ("COMPLETED".equals(transferState.getState()) || "FAILED".equals(transferState.getState()))) {

This is to avoid NPEs

Reply:
Shouldn't it retry if FAILED

Reply:
I wanted to ask should it come out if it fails?. I am sure I can do that too. Since it was the sample program just added that line to close the program gracefully.

Reply:
If the API says it failed, it was failed after retry. So you don't have anything more to do in this transfer. You are good to quit after Failed message